elliott/elliottlib/cli/rhcos_cli.py

```
@@ -136,8 +136,11 @@ def _get_non_layered_tags(runtime, build_id, arch, container_configs, art_repo):
     meta_url = f"{RHCOS_RELEASES_STREAM_URL}/{url_key}/builds/{build_id}/{arch}/meta.json"
 
     LOGGER.info("Fetching RHCOS metadata: %s", meta_url)
-    with request.urlopen(meta_url) as resp:
-        meta = json.loads(resp.read().decode())
+    try:
+        with request.urlopen(meta_url) as resp:
```

Ruff (0.15.13)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)